src/reynir_correct/checker.py

```diff
@@ -204,14 +196,18 @@ class GreynirCorrect(Greynir):
     _reducer = None
     _lock = Lock()
 
-    def __init__(self, settings: Settings, pipeline: CorrectionPipeline, **options: Any) -> None:
-        self._annotate_unparsed_sentences = options.pop("annotate_unparsed_sentences", True)
-        self._ignore_rules: FrozenSet[str] = options.get("ignore_rules", set())
+    def __init__(
+        self,
+        settings: Settings,
+        pipeline: CorrectionPipeline,
+        annotate_unparsed_sentences: bool,
```

Comment:
Perhaps consider whether some of these should be keyword-only arguments? That could help to keep the API stable as more options might be added in the future.

Reply:
Resolved by removing the flag entirely, now we are only accepting `**options`, which are passed down to other modules.